tests/lib/tools/tests.env

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
```

Comment:
I would love to see some introduction text to this tool that explains the purpose/why this is used, otherwise looks great

Reply:
Agree. I see some tools add more explanation in show_help() e.g. `repack-kernel`.